internal/cmd/internal/olmv1/printing.go

```diff
@@ -0,0 +1,58 @@
+package olmv1
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/duration"
+
+	catalogdv1 "github.com/operator-framework/catalogd/api/v1"
+	olmv1 "github.com/operator-framework/operator-controller/api/v1"
+)
+
+func printFormattedOperators(extensions ...olmv1.ClusterExtension) {
+	tw := tabwriter.NewWriter(os.Stdout, 3, 4, 2, ' ', 0)
+	_, _ = fmt.Fprint(tw, "NAME\tINSTALLED BUNDLE\tVERSION\tSOURCE TYPE\tINSTALLED\tPROGRESSING\tAGE\n")
+	for _, ext := range extensions {
+		age := time.Since(ext.CreationTimestamp.Time)
+		_, _ = fmt.Fprintf(tw, "%s\t%s\t%s\t%s\t%s\t%s\t%s\n",
+			ext.Name,
+			ext.Status.Install.Bundle.Name,
+			ext.Status.Install.Bundle.Version,
+			ext.Spec.Source.SourceType,
+			status(ext.Status.Conditions, olmv1.TypeInstalled),
+			status(ext.Status.Conditions, olmv1.TypeProgressing),
+			duration.HumanDuration(age),
+		)
+	}
+	_ = tw.Flush()
+}
+
+func printFormattedCatalogs(catalogs ...catalogdv1.ClusterCatalog) {
+	tw := tabwriter.NewWriter(os.Stdout, 3, 4, 2, ' ', 0)
+	_, _ = fmt.Fprint(tw, "NAME\tLASTUNPACKED\tSERVING\tAGE\n")
+	for _, cat := range catalogs {
```

Comment:
Now or in a follow-up, I don't have an opinion which:

I would suggest sorting these:
- First by enabled/disabled
- Next by priority
- Next by name

Reply:
And we should include columns for enabled/disable and priority.